### PR TITLE
Add maintain command.

### PR DESCRIPTION
**Pull request by Nonsense331**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ This project attempts to follow [semantic versioning](https://semver.org/)
   * Add config option for default_server directive in nginx.
   * Fixed bug in SSL redirect from 2.0.1
   * Adds ability to gather Ruby, Rails, and apt details from servers and send to a stats collector
-  
+  * Add maintin command
```
Comment (petitJAM) [inline CHANGELOG.md]: typooo

Comment (Nonsense331) [reply CHANGELOG.md]: you're a typo

**Comment by z3ugma** [comment]
🚣

File: CHANGELOG.md (hunk 1 above)
Comment (petitJAM) [reply CHANGELOG.md]: :cry:

**Pull request merged**